fe1-web/model/network/method/message/data/election/CreateElection.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  Hash, Timestamp, Lao,
+} from 'model/objects';
+import { OpenedLaoStore } from 'store';
+import { ProtocolError } from 'model/network/ProtocolError';
+import { validateDataObject } from 'model/network/validation';
+import { ActionType, MessageData, ObjectType } from '../MessageData';
+import { checkTimestampStaleness } from '../Checker';
+
```

Comment:
```suggestion
export interface ElectionQuestion {
     id: string,
     question: string,
     voting_method: string,
     ballot_options: string[],
     write_in: boolean,
   }
```

?

Reply:
This is implemented in the `objects/Election.ts` file